ubuntu-advantage

```diff
@@ -109,6 +111,26 @@ main() {
             service_disable "$service" "$@"
             ;;
 
+        update-fips)
+            local token="$1"
+            local bypass_prompt=0
+            while true ; do
+               case "$2" in
+                  --yes | -y)
+                     bypass_prompt=1
+                     break;
+                     ;;
+                  *)
+                     break;
+                     ;;
+               esac
+            done;
+            service_check_user
+            service_check_support "fips"
```

Comment:
shouldn't we also check that fips is enabled? otherwise you could run `ua update-fips <token>`  when you haven't first enabled it

Reply:
fips_update function takes care of it.  if no initramfs fips package is found on the system "fips_configured" will be 0 and it assumes fips was never enabled on the system and does everything that fips_enable does after installing the updated packages.